R/uvhydrograph-render.R

```diff
@@ -167,6 +167,72 @@ createSecondaryPlot <- function(data, month){
       status_msg <- paste('Corrected data missing for', data$reportMetadata$secondaryParameter)
     }
   } 
+
+  isUpchainSeries <- any(grepl("downsampledUpchainSeries", names(data)))
```

Comment:
We shouldn't need any of this because we want the upchain series to plot onto the reference plot, not make a new plot. Since the upchain series was added to parseUVdata, it should already be in the secondaryData list and will be plot automatically